### PR TITLE
pull request for #54 issue

### PR DESCRIPTION
**Pull request by biakaveron**

### CODE REVIEW

File: classes/Task/Help.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ class Task_Help extends Minion_Task
 	 */
 	protected function _execute(array $params)
 	{
-		$tasks = Minion_Util::compile_task_list(Kohana::list_files('classes/Task'));
+		$tasks = self::compile_task_list(Kohana::list_files('classes/Task'));
```
Comment (zeelot) [inline classes/Task/Help.php]: Could you change this to `Minion_Task::compile_task_list()`? Seems that is our convention.

Comment (biakaveron) [inline classes/Task/Help.php]: Done

**Comment by biakaveron** [comment]
Well, I dont know, where I got a Minion_Task::compile_task_list() method, but there is no such method )) It must be $this->_compile_task_list(). Should I create a new pull request?

PS. Sorry for wasting your time (confused)

**Comment by zeelot** [comment]
No worries, my fault. Fixing this now.

**Pull request merged**